modules/installation-osp-verifying-external-network.adoc

```diff
@@ -0,0 +1,59 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+//
+// DNS resolution KI
+
+[id="installation-osp-verifying-external-network_{context}"]
+= Verifying public network access
+
+OpenStack uses its Neutron networking service to access external networks.
+
+.Prerequisites
+* https://docs.openstack.org/neutron/rocky/admin/config-dns-res.html#case-2-dhcp-agents-forward-dns-queries-from-instances[Configure OpenStack's networking service to have DHCP agents forward instances' DNS queries]
+
+.Procedure
+
+. Using the OpenStack CLI, verify the name and ID of the 'External' network:
++
+----
+$ openstack network list --long -c ID -c Name -c "Router Type"
+
++--------------------------------------+----------------+-------------+
+| ID                                   | Name           | Router Type |
++--------------------------------------+----------------+-------------+
+| 148a8023-62a7-4672-b018-003462f8d7dc | public_network | External    |
++--------------------------------------+----------------+-------------+
+----
+
+A network with an External router type appears in the network list. If at least one doesn't, see https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux_OpenStack_Platform/4/html/Installation_and_Configuration_Guide/Configuring_a_Provider_Network1.html[Create an external network].
+
+[IMPORTANT]
+====
+If the external network's CIDR range overlaps one of the default network ranges, you must change the matching network range by running the installer with a custom `install-config.yaml`. 
```

Comment:
This sounds like a prereq for the default install. (Since you never modify the install-config.yaml in that workflow.)